packages/common/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "common",
```

Comment:
I think this should be `@reapit/common`, so in the future, if we want to publish to `npm`, it will not have naming conflict